File: core/model/Measurement.py
from .SingleMeasurement import SingleMeasurement

class Measurement:
    def __init__(self, time: int | None = None, inside: SingleMeasurement | None = None, outside: SingleMeasurement | None = None):
        self.time = time
        self.inside = inside
        self.outside = outside
        
    def txt(self) -> str:
        if (!self.inside) {
            return f"{self.time}: Keine Daten"
        }
        return f"{self.time}: Innen ({self.inside.txt()})"

File: server/Model/Entity/Measurement.cs
namespace TauLuftAspire.Model.Entity;

public class Measurement
{
    public int Id { get; set; }
    public DateTime Timestamp { get; set; }
    public SingleMeasurement Inside { get; set; }
    public SingleMeasurement Outside { get; set; }
    public bool IsFanRunning { get; set; }

    public DateTime LocalTimestamp => Timestamp.ToLocalTime();

    public double DeltaTp => Inside.DewPoint - Outside.DewPoint;
}
